single_kernel_mongo/core/operator.py

```diff
@@ -248,3 +248,9 @@ def set_permissions(self) -> None:
                     f"{path}",
                 ]
             )
+
+        for path in (
+            self.workload.paths.config_file,
+            self.workload.paths.mongos_config_file,
+        ):
+            self.workload.exec(["chmod", "600", f"{path}"])
```

Comment:
newb questions:

- should the snap do this, given that the snap sets up these files for us? 
- will people using the snap have to do this in order to get `mongod`/`mongos` working from the start? If so then this should be moved to the snap

Reply:
They won't have to do it no.
This is purely a question of security in this case because it will contains some secrets that we want to allow only the owner to be able to read.
In the other cases it's not needed.

Reply:
Good Explanation - thank you!